test/inductor/test_flex_attention.py

```diff
@@ -989,14 +989,15 @@ def composed_score_mod(score, b, h, m, n):
         self.run_test(composed_score_mod, dtype)
 
     @supported_platform
+    # @expectedFailure  # TODO: Remove this after supporting compiled flex attention with training bias
```

Comment:
I dont really trust this...

Reply:
You don't believe in expectedFailure?

Reply:
This is why I didnt trust this:

https://github.com/pytorch/pytorch/pull/137526/files#diff-e3963412cc249e81fecfcf8774f5428b2b5e837ff3633ae13d8b7886ab5bc3b9R2074